extensions/raycast-mux/package.json

```diff
@@ -0,0 +1,136 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "raycast-mux",
+  "type": "module",
+  "title": "Mux.com",
+  "description": "Interact with your Mux.com assets",
+  "icon": "extension-icon.png",
+  "author": "adam_engebretson",
+  "repository": "github:adamgoose/raycast-mux",
+  "license": "MIT",
+  "categories": [
+    "Developer Tools",
+    "Media"
+  ],
+  "commands": [
+    {
+      "name": "get-playback-id",
+      "title": "Get Playback ID",
+      "subtitle": "Convert an Asset ID to a Playback ID",
```

Comment:
```suggestion
      "subtitle": "Mux.com",
```

Reply:
The conversion between Asset IDs and Playback IDs is the value that this (and its opposite partner) bring; I really would like to provide this as a subtitle. Would it be better if it said "Convert a Mux.com Asset ID to a Playback ID," so to keep the additional context and include the extension name?